app/services/id_token_builder.rb

```diff
@@ -36,16 +36,19 @@ def jwt_payload
   end
 
   def id_token_claims
-    {
-      acr: (acr if !sp_requests_vot?),
-      vot: (vot if sp_requests_vot?),
-      vtm: (IdentityConfig.store.vtm_url if sp_requests_vot?),
+    claims = {
       nonce: identity.nonce,
       aud: identity.service_provider,
       jti: SecureRandom.urlsafe_base64,
       at_hash: hash_token(identity.access_token),
       c_hash: hash_token(code),
-    }.compact
+    }
+
+    if !sp_requests_vot?
+      claims[:acr] = acr
+    end
+
+    claims.compact
```

Comment:
this `compact` is now redundant since we don't seem to be adding anything that could be nil?
```suggestion
    claims
```